setup.cfg

```diff
@@ -110,6 +110,9 @@ asdf.resource_mappings =
 asdf.extensions =
   sunpy = sunpy.io.special.asdf.entry_points:get_extensions
 
+[bdist_wheel]
+py_limited_api = cp38
```

Comment:
Put a comment here saying if you change this you need to change the line in `_pyana.c` or else we *will* forget.

Reply:
Silly question, how does one get the version hex? Is it just a conversion of the number to hexadecimal?

Reply:
It's defined here: https://docs.python.org/3.12/c-api/apiabiversion.html

And more general details here: https://docs.python.org/3/c-api/stable.html#c.Py_LIMITED_API